translations/german/helptickets.ts

```diff
@@ -139,7 +139,7 @@ export const translations: Translations = {
 		"Help Ticket": "Help-Ticket",
 		"Issue": "Problem",
 		"A Global Staff member will be with you shortly.": "Ein globales Staffmitglied wird gleich da sein.",
-		"${this.inputUsername} does not have an open ticket.": "${this.inputUsername} besitzt kein aktives Ticket.",
+		"${inputUsername} does not have an open ticket.": "${inputUsername} besitzt kein aktives Ticket.",
```

Comment:
You don't have to change the stuff in the `${}`; template strings just work positionally AFAICT. (I tested this in <<croutons>> if you want to see what I mean.)

Reply:
O

Reply:
They look better without the `this.` anyway.

Reply:
They look better without the `this.` anyway.